openstack/cdn/v1/base/requests_test.go

```diff
@@ -0,0 +1,45 @@
+package base
+
+import (
+	"testing"
+
+	th "github.com/rackspace/gophercloud/testhelper"
+	fake "github.com/rackspace/gophercloud/testhelper/client"
+)
+
+func TestGetHomeDocument(t *testing.T) {
+	th.SetupHTTP()
+	defer th.TeardownHTTP()
+	HandleGetSuccessfully(t)
+
+	actual, err := Get(fake.ServiceClient()).Extract()
+	th.CheckNoErr(t, err)
+
+	expected := HomeDocument{
+		"rel/cdn": `{
+        "href-template": "services{?marker,limit}",
+        "href-vars": {
+            "marker": "param/marker",
+            "limit": "param/limit"
+        },
+        "hints": {
+            "allow": [
+                "GET"
+            ],
+            "formats": {
+                "application/json": {}
+            }
+        }
+    }`,
```

Comment:
Hmm. The home document isn't parsed the whole way? This looks like it's left mostly unparsed as a `map[string]string`.